### PR TITLE
LPS-102602 Autogenerated.

### PR DESCRIPTION
**Pull request by cgoncas**

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
Just started reviewing :)

File: modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/portlet/action/EditLayoutMVCActionCommand.java [hunk 1]
```diff
@@ -149,10 +149,25 @@ protected void doProcessAction(
...
+
+		long openGraphImageFileEntryId = ParamUtil.getLong(
+			actionRequest, "openGraphImageFileEntryId");
 
 		_layoutSEOEntryService.updateLayoutSEOEntry(
 			groupId, privateLayout, layoutId, useCustomCanonicalURL,
-			canonicalURLMap, serviceContext);
+			canonicalURLMap, useCustomDescription, openGraphDescriptionMap,
+			useCustomTitle, openGraphTitleMap, openGraphImageFileEntryId,
+			serviceContext);
```
Comment (adolfopa) [inline modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/portlet/action/EditLayoutMVCActionCommand.java]: Look for "String title" in the portal. In method signatures, the title always comes immediately before than the description, so we should do the same here. E.g. doFoo(useDescription, descriptionMap, useTitle, titleMap) ==> doFoo(useTitle, titleMap, useDescription, description)

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/seo.jsp [hunk 2]
```diff
@@ -114,7 +114,103 @@ UnicodeProperties layoutTypeSettings = selLayout.getTypeSettingsProperties();
...
+
+				<aui:input id="openGraphImageFileEntryId" name="openGraphImageFileEntryId" type="hidden" />
+			</div>
+		</c:when>
+		<c:otherwise>
+			<div id="<portlet:namespace />openGraphSettings">
+				<aui:input checked="<%= false %>" helpMessage="use-custom-open-graph-title-help" id="useCustomTitle" label="use-custom-title" name="openGraphTitle" type="checkbox" />
+
+				<aui:input label="<%= StringPool.BLANK %>" localized="<%= true %>" name="openGraphTitle" type="text">
+				</aui:input>
```
Comment (adolfopa) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/seo.jsp]: Can we remove the closing </aui:input>? All other inputs that have no value there are autoclosing.

File: modules/apps/layout/layout-seo-service/service.xml [hunk 3]
```diff
@@ -25,9 +25,14 @@
 
 		<column name="privateLayout" type="boolean" />
 		<column name="layoutId" type="long" />
-		<column name="enabled" type="boolean" />
+		<column name="enabledCanonicalURL" type="boolean" />
 		<column localized="true" name="canonicalURL" type="String" />
+		<column name="enabledOpenGraphTitle" type="boolean" />
+		<column localized="true" name="openGraphTitle" type="String" />
+		<column name="enabledOpenGraphDescription" type="boolean" />
```
Comment (adolfopa) [inline modules/apps/layout/layout-seo-service/service.xml]: enabledCanonicalURl ==> canonicalURLEnabled
enabledOpenGraphTitle ==> openGraphTitleEnabled
etc.

**Pull request closed without merging**